main.go

```diff
@@ -200,6 +200,13 @@ func addWorktreeAndSwap(gitRoot, dest, branch, rev string) error {
 	}
 	log.Printf("fetched origin/%s", branch)
 
+	// find the real commit for the rev
+	output, err := runCommand(gitRoot, "git", "rev-list", "-n1", rev)
```

Comment:
will do

On Tue, Sep 13, 2016 at 5:05 PM, Mike Danese <notifications@github.com>
wrote:

> In main.go
> <https://github.com/kubernetes/git-sync/pull/2#discussion_r78667253>:
>
> > @@ -200,6 +200,13 @@ func addWorktreeAndSwap(gitRoot, dest, branch, rev string) error {
> >  	}
> >  	log.Printf("fetched origin/%s", branch)
> >
> > +	// find the real commit for the rev
> > +	output, err := runCommand(gitRoot, "git", "rev-list", "-n1", rev)
>
> Check error?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/git-sync/pull/2/files/7d774b8b52a75fff8ddc5435b9371c0cd7be222b..528f4eadc9f8aa686cec8e4f61a46d358a1609cd#r78667253>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFVgVN5XyMomZ2aDsnt66LTpqkW5-qT8ks5qpzo5gaJpZM4Jssne>
> .
>